### ISSUE TITLE
currentProgramDateTime is undefined on Android

### ISSUE THREAD

**Issue by bpkmatthiasv** [THEOplayer/react-native-theoplayer#118]
When listening to the `PlayerEventType.TIME_UPDATE` event, property `currentProgramDateTime` is `undefined` on Android.

Using react-native-theoplayer": "^2.5.0"

Stream: https://tagesschau.akamaized.net/hls/live/2020115/tagesschau/tagesschau_1/master.m3u8

iOS:
{"currentProgramDateTime": 1683017551409.3333, "currentTime": 7188009.333333333, "date": 2023-05-02T08:52:45.167Z, "type": "timeupdate"}

Web:
DefaultTimeupdateEvent {type: 'timeupdate', date: Tue May 02 2023 10:53:11 GMT+0200 (Central European Summer Time), currentTime: 7189027.979, currentProgramDateTime: 1683017576427} 

Android:
{"currentProgramDateTime": undefined, "currentTime": 7188273, "date": 2023-05-02T08:54:01.091Z, "type": "timeupdate"}

**Comment by tvanlaerhoven** [comment]
@bpkmatthiasv Thanks for reporting this. There's indeed an issue in our Android SDK currently, resulting in the `currentProgramDataTime` property to always `null` when dispatching a `timeupdate` event.
A fix will be part of the next Android SDK (5.0.4).

In addition, the formatting of this field was wrong for Android: https://github.com/THEOplayer/react-native-theoplayer/pull/119/

**Comment by tvanlaerhoven** [comment]
@bpkmatthiasv the issue was resolved in Android player SDK v5.2.0, so make sure to target the latest version here, or explicitly target it in `gradle.properties`:

```
# Version of the THEOplayer SDK, if not specified, the latest available version is set.
THEOplayer_sdk=5.2.0
```

**Comment by bpkmatthiasv** [comment]
The fix is working, thanks !
